tests/integration/helpers.py

```diff
@@ -9,19 +9,26 @@
 # Ignore functions having too many arguments (will be removed in the future)
 # pylint: disable=too-many-positional-arguments
 
+# Ignore too many args warning. I NEED THEM OKAY ?!
+# pylint: disable=too-many-arguments
```

Comment:
I think you can also disable this for all test files in the pylint configuration, I think we reached that agreement in The Hague